src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -407,14 +407,19 @@ def evaluate_from_recipe(cls, cfg: DictConfig) -> Tuple[nn.Module, Tuple]:
 
         return model, valid_metrics_dict
 
+    @classmethod
+    def evaluate_from_recipe(cls, cfg: DictConfig) -> Tuple[nn.Module, Tuple]:
```

Comment:
This is not how we deprecate methods in SG.
See super_gradients.common.deprecate.deprecated.

Reply:
Changed,  [ccd7302](https://github.com/Deci-AI/super-gradients/pull/1904/commits/ccd73021509e912577aa3f094aac2754b190db7b)